README.md

```diff
@@ -1,2 +1,25 @@
-# hw01
-Repository for homework 01
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# HOMEWORK 1
+
+__*Hello*__
+
+My name is Elaine Chen. I'm a third-year undergraduate student at the University of Chicago studying Political Science and Economics.
+
+![Portrait](https://raw.githubusercontent.com/elaineywchen/hw01/master/portrait%20for%20r.jpg){width=250px}
+
+##Where I'm from
+
+I moved around growing up. I was born in Taiwan, and then at three years old, I moved to the Bay Area. I stayed in the Bay Area until the end of middle school, and then I moved back to Asia to Beijing. I stayed in Beijing until high school graduation, and then I came to Chicago for college.
+
+##What I do
```

Comment:
Also, maybe you can apply headings to this. Make sure it is in the correct format.